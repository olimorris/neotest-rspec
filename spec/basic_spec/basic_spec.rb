RSpec.describe 'Maths' do
  it 'adds two numbers together' do
    expect(2 + 2).to eq(4)
  end
  it 'subtracts two numbers together' do
    expect(3 - 2).to eq(1)
  end
end
