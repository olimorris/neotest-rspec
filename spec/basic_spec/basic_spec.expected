./SPEC/BASIC_SPEC/BASIC_SPEC.START.RB@@PASSED@@adds two numbers together
